<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="ep_list">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_pattern">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="prj_comboBox">
           <item>
            <property name="text">
             <string>woody</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>woowoo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nay</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="folders_pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Make Folders</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="update_sg_ep_pushButton">
           <property name="text">
            <string>Upd EP in SG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="proc_comboBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>By Time</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>comp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>render</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>anim</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>layout</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Premaster</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="renderSeq">
           <property name="text">
            <string>Render Seq</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_forceupd">
           <property name="text">
            <string>Force Upd</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_updateseq">
           <property name="text">
            <string>Update Seq</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="debugButton">
           <property name="text">
            <string>Debug</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_ep_info">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="indent">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="logOutput">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_button">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
